tests/scripts/thread-cert/border_router/test_vicarious_router_solicit.py

```diff
@@ -1,216 +0,0 @@
-#!/usr/bin/env python3
-#
-#  Copyright (c) 2021, The OpenThread Authors.
-#  All rights reserved.
-#
-#  Redistribution and use in source and binary forms, with or without
-#  modification, are permitted provided that the following conditions are met:
-#  1. Redistributions of source code must retain the above copyright
-#     notice, this list of conditions and the following disclaimer.
-#  2. Redistributions in binary form must reproduce the above copyright
-#     notice, this list of conditions and the following disclaimer in the
-#     documentation and/or other materials provided with the distribution.
-#  3. Neither the name of the copyright holder nor the
-#     names of its contributors may be used to endorse or promote products
-#     derived from this software without specific prior written permission.
-#
-#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS 'AS IS'
-#  AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
-#  IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
-#  ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
-#  LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
-#  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
-#  SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
-#  INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
-#  CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
-#  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
-#  POSSIBILITY OF SUCH DAMAGE.
-#
-import logging
-import unittest
-from ipaddress import IPv6Network
-
-import config
-import thread_cert
-
-# Test description:
-#   This test verifies Vicarious Router Solicitation.
-#
-# Topology:
-#    -------------(eth)----------------------------
-#           |               |           |
-#          BR1             BR2         HOST
-#           |               |
-#        ROUTER1         ROUTER2
-#
-#     Thread Net1       Thread Net2
-#
-
-BR1 = 1
-ROUTER1 = 2
-BR2 = 3
-ROUTER2 = 4
-HOST = 5
-
-CHANNEL1 = 18
-CHANNEL2 = 19
-
-
-class MultiThreadNetworks(thread_cert.TestCase):
```

Comment:
If we leave the test alone, it can still pass.

The test seems unrelated to `OPENTHREAD_CONFIG_BORDER_ROUTING_VICARIOUS_RS_ENABLE` from what it does. 

Maybe we can simply keep and rename the test? @jwhui @wgtdkp

Reply:
I renamed this test to `test_on_link_prefix.py`. Thoughts?

Reply:
SGTM.